microsetta_admin/server.py

```diff
@@ -556,6 +640,125 @@ def new_kits():
                          mimetype='text/csv')
 
 
+def read_csv_file(file):
+    content = file.read().decode('utf-8-sig')
+    return [row[0] for row in csv.reader(io.StringIO(content),
+                                         skipinitialspace=True)
+            if row]
+
+
+def handle_api_request(payload):
```

Comment:
It doesn't seem appropriate to have a generic name on a function that only handles a very specific API path. Can you either adjust the function name, or add the API path as an argument? Also, helper functions should be preceded by an underscore (`_handle_api_request`).